editor/src/clj/dynamo/graph.clj

```diff
@@ -599,6 +599,13 @@
   (assert node-id)
   (it/update-property node-id p f args))
 
+(defn update-property-ec
+  "Same as update-property, but injects the in-transaction evaluation-context
+  as the first argument to the update-fn."
+  [node-id p f & args]
+  (assert node-id)
+  (it/update-property-ec node-id p f args))
+
```

Comment:
I have a situation where I need the *in-transaction* value of the `:legacy-scale` since we're doing all of this at load-time. This is an implementation of `g/update-property` that injects the *in-transaction* `evaluation-context` as the first argument to the `update-fn`.